Vagrantfile

```diff
@@ -5,8 +5,12 @@
 VAGRANTFILE_API_VERSION = "2"
 
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
-    config.vm.box = "lvillani/win2008r2"
+    config.vm.box = "modernIE/w10-edge"
```

Comment:
Looks good, I will probably change this VM to use `modernIE/w7-ie8` since it weighs less and should be fine for our purposes.